code/modules/paperwork/contract.dm

```diff
@@ -0,0 +1,140 @@
+#define CONTRACT_POWER 1
+#define CONTRACT_WEALTH 2
+#define CONTRACT_PRESTIGE 3
+#define CONTRACT_MAGIC 4
+#define CONTRACT_SLAVE 5
+#define CONTRACT_UNWILLING 6
+
+
+/* For employment contracts and demonic contracts */
+
+/obj/item/weapon/paper/contract
+	throw_range = 3
+	throw_speed = 3
+	var/signed = 0
+	var/mob/living/target
+
+/obj/item/weapon/paper/contract/proc/update_text()
+	name = "paper- generic contract"
+
+
+
+/obj/item/weapon/paper/contract/employment/New(mob/living/nOwner)
+	..()
+	target = nOwner
+	update_text()
+
+
+/obj/item/weapon/paper/contract/employment/update_text()
+	name = "paper- [target] employment contract"
+	info = "<center><B>Official copy of Nanotransen employment agreement</B></center><BR><BR><BR>Contract for [target]<BR><BR><BR>Placeholder text"
+
+
+/obj/item/weapon/paper/contract/employment/attack(mob/living/M, mob/living/carbon/human/user)
+	var/deconvert = 0
+	if((M == target) && !target.hasSoul)
+		if(user.mind && (user.mind.assigned_role == "Lawyer"))
+			deconvert = prob (25)
+		if (user.mind && (user.mind.assigned_role =="Head of Personnel") || (user.mind.assigned_role == "Centcom Commander"))
+			deconvert = prob (10) // the HoP doesn't have AS much legal training
+	if(deconvert)
+		M.visible_message("<span class='notice'>[user] reminds [M] that [M]'s soul was already purchased by Nanotransen!</span>")
+		M << "<span class='boldnotice'>You feel that your soul has returned to it's rightful owner, Nanotransen.</span>"
+		M.returnSoul()
+		M.removeDemonBoons()
+		return
+	else
+		if(ishuman(M) && !istype(M:head, /obj/item/clothing/head/helmet))
```

Comment:
The colon operator ```:``` is not allowed, please typecast and use the dot operator ```.``` instead.